### PR TITLE
Profile page logic

### PR DESCRIPTION
**Pull request by MertzAndreas**
## Description

Added UI changes such as icon button animations, also partially implemented hook and API interaction for organisation interactions. Create Org works but rest is implemented in a different page.

## Type of PR

- Feature
- Refactor

## Related Issues/Tickets

#132 (Partial as @MaksimHamko creates the view for invitations)

## Checklist

- [] Completed the user story described by the issue (if applicable)
- [X] Code follows the code guidelines
- [X] Tests have been added/updated (if applicable)

### CODE REVIEW

File: apis/organisationAPI.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+): Promise<OrgDTO> => {
+  const params = new URLSearchParams();
+  params.append("id", userId);
+  const url = `${BASE_URL}/organizations?${params.toString()}`;
+  const res = await fetch(url, {
+    method: "POST",
+    headers: { "Content-Type": "application/json" },
+    body: JSON.stringify({ name: orgName }),
+  });
+  if (!res.ok) throw new Error("Kunne ikke slette organisation");
```
Comment (Tbjerregaard) [inline apis/organisationAPI.ts:44]: "Kunne ikke oprette organisation"*

**Review by Tbjerregaard — CHANGES_REQUESTED** [review]
Overall good only found one small typo. One change that could be nice would be when the user is creating an organisation that they would be able to cancel it a bit more intuitively/easier . The action of swiping it back down is not very user friendly imo, at least I experienced a skill issue for a bit.

**Review by DanielRBech — DISMISSED** [review]
I like - well done 💯

File: __tests__/useOrganisation.test.tsx [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+    addToast: jest.fn(),
+  }),
+}));
+
+jest.mock("../providers/AuthenticationProvider", () => ({
+  useAuthentication: () => ({
+    userId: "mockUserId",
+  }),
+}));
+
```
Comment (Tbjerregaard) [inline __tests__/useOrganisation.test.tsx:38]: this is nice

**Review by Simo205j — APPROVED** [review]

**Pull request merged**